examples/set-default-labels-nested-package/app2_mysql/README.md

```diff
@@ -0,0 +1,21 @@
+# app2
+
+## Description
+sample description
+
+## Usage
+
+### Fetch the package
+`kpt pkg get REPO_URI[.git]/PKG_PATH[@VERSION] app2`
+Details: https://kpt.dev/reference/cli/pkg/get/
+
+### View package content
+`kpt pkg tree app2`
+Details: https://kpt.dev/reference/cli/pkg/tree/
+
+### Apply the package
+```
+kpt live init app2
+kpt live apply app2 --reconcile-timeout=2m --output=table
```

Comment:
If we want to keep this README, we should follow the styling guide as other docs.

Reply:
we only need the README in root package. cleaned up this one.